source/adl-standalone/tests/getMore.yml

```diff
@@ -0,0 +1,34 @@
+collection_name: &collection_name "quux"
+database_name: &database_name "test"
+namespace: &namespace "test.quux"
```

Comment:
`namespace` is neither used in this test file, nor is it defined anywhere in the CRUD spec test format. Please strike this from any test file in this PR.